internal/handler/msg_createuser.go

```diff
@@ -29,6 +37,116 @@ func (h *Handler) MsgCreateUser(ctx context.Context, msg *wire.OpMsg) (*wire.OpM
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/1491
-	return nil, common.Unimplemented(document, document.Command())
+	dbName, err := common.GetRequiredParam[string](document, "$db")
+	if err != nil {
+		return nil, err
+	}
+
+	if dbName != "$external" && !document.Has("pwd") {
```

Comment:
We don't handle `$external` in the current issue, right? Do we need a separate issue to support it?

Reply:
We don't handle it at this current issue. I think this will be handled by #3777, #3778, and #3784 if I'm not mistaken.

Reply:
Let's add a `TODO` then, so we don't miss it ;)

Reply:
Added.